graphstorm-processing/scripts/run_distributed_processing.py

```diff
@@ -45,8 +45,11 @@
     Number of EC2 instances to use for the processing job. (Default: 2)
 --instance-type: str
     EC2 instance type for the processing job. (Default: 'ml.r5.4xlarge')
---add-reverse-edges: bool
-    When set to True, will create reverse edges for every edge type. (Default: True)
+--add-reverse-edges: str
+    When set to "True", will create reverse edges for every edge type. (Default: "True")
```

Comment:
Why not use bool?

Reply:
Just consistency, the entry point scripts for SageMaker need to use String values, so we do the same here. 

I can change to to a flag for the launcher if you think that makes more sense.